content/general/concepts/what-is-an-object/what-is-an-object.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'What is an Object?'
+Description: 'An Object is an instance of a class. It is created when we instantiate the class. Object contains the properties and functionalities. When an Object is created memory will be allocated for it.Creating an Object in Java:
+class Employee{
+  int ssn;
+  String name;
+  float age;
+  public Employee(){}
+}Employee emp1=new Employee();
+In the above exmaple we created an object for the class ***Employee***. In Java we create a new object using new keyword. The new keyword tells the compiler to create a new instance of a class and allocate memory for it.
+'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Objects'
+  - 'Inheritance'
+  - 'Methods'
+  - 'Structure'
+  - 'Encapsulation'
+  - 'Classes'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+  - 'paths/front-end-engineer-career-path'
+---
+
+An Object is an instance of a class. It is created when we instantiate the class. Object contains the properties and functionalities. When an Object is created memory will be allocated for it.
+
+Creating an Object in Java:
+
+```
+class Employee{
+  int ssn;
+  String name;
+  float age;
+  public Employee(){}
+}
+
+Employee emp1=new Employee();
+
+
+```
+
+In the above example we created an object for the class ***Employee***. In Java we create an object using the **new** keyword. The **new** keyword tells the compiler to create a new instance of a class and allocate memory for it.
```

Comment:
In the above example, we created five different objects from the class `Employee`.